jmeter/plans/apply.rb

```diff
@@ -2,6 +2,7 @@
 
 BASEURL = ENV.fetch('JMETER_TARGET_BASEURL')
 WAIT_FACTOR = ENV.fetch('JMETER_WAIT_FACTOR', 1).to_f
+RAMPUP = ENV.fetch('JMETER_RAMPUP', 0).to_i
```

Comment:
The ruby-jmeter gem (or maybe JMeter itself, it's unclear to me currently) appears to set a default ramp up of half the thread count if you don't specify one explicitly. Would it make sense to set some kind of default ramp up value here? Maybe 60 or so?